Make fails

Running just `make` fails with: 
```
make: *** No rule to make target '.npminstall', needed by 'check-style'. Stop.
```

I think, https://github.com/mattermost/mattermost-plugin-github/blob/master/Makefile#L8 should be changed to `check-style: 
webapp/.npminstall gofmt`

--- a/.gitignore
+++ b/.gitignore
@@ -13,15 +13,13 @@
 # Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
 .glide/
 
-# Build                                                                                                                                 
-/coverage.txt                                                                                                                           
-/plugin.exe                                                                                                                             
-/mattermost-github-plugin*.tar.gz                                                                                                         
-/dist                                                                                                                                   
-server/vendor                                                                                                                           
-node_modules                                                                                                                            
-.npminstall                                                                                                                             
-webapp/dist 
+# Build
+/dist
+server/coverage.txt
+server/vendor
+webapp/.npminstall
+webapp/node_modules
+webapp/dist
 
 *.swp
 

--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ GOARCH=amd64
 
 .PHONY: build test run clean stop check-style gofmt
 
-check-style: .npminstall gofmt
+check-style: webapp/.npminstall gofmt
 	@echo Checking for style guide compliance
 
 	cd webapp && npm run check
@@ -27,7 +27,7 @@ gofmt:
 	done
 	@echo "gofmt success"; \
 
-test: .npminstall
+test: webapp/.npminstall
 	@echo Not yet implemented
 
 webapp/.npminstall:
@@ -82,7 +82,7 @@ localdeploy: dist
 	rm -rf ../mattermost-server/plugins/github
 	tar -C ../mattermost-server/plugins/ -zxvf ../mattermost-server/plugins/mattermost-github-plugin-$(GOOS)-$(GOARCH).tar.gz
 
-run: .npminstall
+run: webapp/.npminstall
 	@echo Not yet implemented
 
 stop:

--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ gofmt:
 	@echo "gofmt success"; \
 
 test: webapp/.npminstall
-	@echo Not yet implemented
+	cd server && go test -v -coverprofile=coverage.txt ./...
 
 webapp/.npminstall:
 	@echo Getting dependencies using npm

